templates/party/add_party_from_index.html.twig

```diff
@@ -0,0 +1,30 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
```

Comment:
Does this need a translation domain?

Reply:
Added translation domain and made translations